debian/debathena-pyhesiodfs.service

```diff
@@ -0,0 +1,20 @@
+# Author: Lizhou Sha <slz@mit.edu>
+# This file is part of the debathena-pyhesiodfs package.
+
+[Unit]
+Description=Hesiod automounter for Athena lockers
+Requires=openafs-client.service
+After=local-fs.target network.target openafs-client.service
+Before=umount.target
```

Comment:
openafs-client needs to be Before unmount.target because openafs needs to do special handling to actually do the unmount cleanly and unload the kernel module; without that dependency systemd would just do 'umount /afs', the unit's shutdown entries would not get run, and we'd be stuck with the kernel module loaded.

with respect to remote-fs.target, there is an argument that we should be before it, since we do provide a filesystem namespace that other things might use.